tests/v2/validation/nodescaling/replace.go

```diff
@@ -48,22 +50,24 @@ func ReplaceNodes(t *testing.T, client *rancher.Client, clusterName string, isEt
 	require.NoError(t, err)
 	numOfNodesBeforeDeletion, nodeToDelete := MatchNodeToRole(t, client, clusterID, isEtcd, isControlPlane, isWorker)
 
-	machineToDelete, err := client.Steve.SteveType(machineSteveResourceType).ByID("fleet-default/" + nodeToDelete.Annotations[machineSteveAnnotation])
-	require.NoError(t, err)
+	for i := range nodeToDelete {
```

Comment:
this is now a list. Isn't it replacing every matching node now instead of just 1?

Reply:
Yeah, I can just change the name to `nodesToDelete`.